src/documentation/install.page

```diff
@@ -8,32 +8,32 @@ sort_info: 2
 lang: en
 ---
 
-# Install Uniqush System
+# Installing Uniqush and its Dependencies
 
-## Install uniqush-push
+## Installing uniqush-push
 
-uniqush-push now is the only component in the Uniqush System.
+uniqush-push now is the only component in Uniqush.
 
 It is higly recommanded to install from **binary** package on **64-bit** system.
 
 ### Install Redis
 
```

Comment:
Redis should be installed on a server which has low latency to the server uniqush-push is installed on (e.g. the same datacenter). One solution is to install both programs on the same server.